liquibase-integration-tests/src/test/java/liquibase/dbtest/AbstractIntegrationTest.java

```diff
@@ -1310,6 +1308,27 @@ public void verifyIndexIsCreatedWhenAssociatedWithPropertyIsSetAsNone() throws D
 
     }
 
+    @Test
+    public void makeSureNoErrorIsReturnedWhenTableNameIsNotSpecified() throws DatabaseException {
+        // This is a test to validate most of the DBs do not throw an error when tableName is not
+        // specified as part of primaryKeyExistsPrecondition.
+        clearDatabase();
+        String errorMsg = "";
+        try {
+            runUpdate("changelogs/common/preconditions/preconditions.changelog.xml");
+        }catch(CommandExecutionException e) {
+            errorMsg = e.getMessage();
+        }
+        finally {
+            clearDatabase();
+        }
+
+        if(!(database instanceof H2Database || database instanceof MySQLDatabase || database instanceof HsqlDatabase
+                || database instanceof SQLiteDatabase || database instanceof DB2Database)) {
+            Assert.assertTrue(errorMsg.isEmpty());
+        }
```

Comment:
Hi @MalloD12 ! Instead of replicating the test code 5 times, would not be better to add an validation here for the expected error message too?

Reply:
Hi @filipelautert, I wouldn't add to the same test to validate two possible scenarios in the same test, but yes I can add a new test with a similar if logic and in that case assert error message is present in that case.

Reply:
Nice, otherwise if the message is changed we wil need to fix 5 tests instead of just one ;)

Reply:
Done